util/localconfig/localconfig.go

```diff
@@ -303,3 +313,11 @@ func GetUsername(subject string) string {
 	}
 	return subject
 }
+
+func GetFilePermission(fi os.FileInfo) error {
+	//argo directory should have mode 0700 & file should have permission -  0600.
+	if fi.Mode().Perm() == 0600 {
```

Comment:
As discussed on call, I think we should allow 0400 in case someone wants to further lock down a config file (maybe in an automated environment).

Reply:
i would prefer this, What do you say?

`	if fi.Mode().Perm() != 0600 || fi.Mode().Perm() != 0400 {
		return fmt.Errorf("config file has incorrect permission flags: %s  "+
			"change the permission either to 0400 or 0600 permission", fi.Mode().Perm().String())
	}
	return nil`

Reply:
make sense